FND/Relations/Relations.rdf

```diff
@@ -103,8 +102,7 @@ the FIBO FND RFC, was revised in advance of the September 2013 New Brunswick, NJ
 	<owl:ObjectProperty rdf:about="&fibo-fnd-rel-rel;characterizes">
 		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-rel-rel;refersTo"/>
 		<rdfs:label>characterizes</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-fnd-rel-rel;Reference"/>
-		<skos:definition>a feature or quality that distinguishes something from something else</skos:definition>
+		<skos:definition>describes the nature, features or quality of</skos:definition>
```

Comment:
Too broad to be useful. The giveaway is the "or" in the definition - maybe we should have a hygiene test for that!

It's impossible from these definitions to decide whether to use appliesTo or characterizes. Seriously.
We have Catalog characterizes Product; why would it not be appliesTo?
Given that characterizes is used with Aspect (which is at least coherent) then, simce Catalog is nothing at all like an Aspect, that Catalog usage is wrong.

qualifies is subPropertyOf characterizes but with quite different, rather than more specific meaning "limits, constrains, or refines". Again not coherent.

Another usage is that a specific CodeScheme characterizes a CodeRepository. I wouldn't say that it "describes the quality of" it. Again not coherent.

Reply:
It is not too broad to be useful - that's the definition of the term 'characterizes'.  I agree that some of the places where it is used should be corrected, such as in the case of a catalog, and will make that change.  Applies to is fine for that case and for the code scheme case, which I will revise accordingly.

Reply:
Raised FND-311 to review uses of appliesTo; added FND-312 to replace hasDescription with isDescribedBy